.github/workflows/k8s-integration.yaml

```diff
@@ -17,9 +17,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - shell: bash
+        run: |
+          echo "loading env from .makerc"
+          while IFS='' read -r LINE || [ -n "${LINE}" ]; do
+            echo "$LINE" >> $GITHUB_ENV;
+          done < .makerc
```

Comment:
`cat .makerc >> "$GITHUB_ENV`?

Reply:
that won't work sadly.
changed to use `c-py/action-dotenv-to-setenv@v3` instead